examples/lock-app/k32w/main/include/AppTask.h

```diff
@@ -81,6 +81,7 @@ class AppTask
 
     Function_t mFunction;
     bool mResetTimerActive;
+    bool mSyncClusterToButtonAction;
```

Comment:
where mSyncClusterToButtonAction gets initialized?

Reply:
I had the impression that mSyncClusterToButtonAction  is zero initialized (false) when the class constructor is called. Is this true?

Reply:
It is not guaranteed since this member variable  is not static, you can use either initialization list or the constructor.

With C++11, you can also provide an initializer in the declaration for non-static member variables, such as:

class AppTask {
   bool mSyncClusterToButtonAction = false;
};

Reply:
Thanks for the recommendations, explicitly initialized the member class.